src/Tizen.NUI/src/public/InputMethod.cs

```diff
@@ -387,7 +391,14 @@ private PropertyMap ComposingInputMethodMap()
         {
             PropertyMap _outputMap = new PropertyMap();
             if (_panelLayout != null) { _outputMap.Add("PANEL_LAYOUT", new PropertyValue((int)_panelLayout)); }
-            if (_actionButton != null) { _outputMap.Add("BUTTON_ACTION", new PropertyValue((int)_actionButton)); }
+            if (_actionButton != null)
+            {
+                // Temporarily specify the values to match the types of ecore_imf.
+                if (_actionButton == InputMethod.ActionButtonTitleType.Search) _actionButton = (InputMethod.ActionButtonTitleType.Search - 1); // 6
+                else if (_actionButton == InputMethod.ActionButtonTitleType.Send) _actionButton = (InputMethod.ActionButtonTitleType.Send - 1); // 7
+                else if (_actionButton == InputMethod.ActionButtonTitleType.SignIn) _actionButton = (InputMethod.ActionButtonTitleType.SignIn - 1); // 8
+                _outputMap.Add("BUTTON_ACTION", new PropertyValue((int)_actionButton));
```

Comment:
Is it unnecessary to add an exception for the Unspecified/None??

Reply:
Yes, you're right.
I'll update the patch for these exceptions.